src/components/App.js

```diff
@@ -1,6 +1,8 @@
 import React from 'react';
 import Display from './display';
 import ButtonPanel from './buttonpanel';
+// eslint-disable-next-line no-unused-vars
```

Comment:
With the ESLint inline comment, we are indirectly changing the rules in `.eslintrc.json` by disabling these rules, let's fix the codebase instead.

Perhaps you're using this to pass the linter's check because of the `Calculate` you import, kindly remove it in the next milestone.

Reply:
yes??  it was used to pass linters check. And would be removed in the next milestone